### PR TITLE
doc: Make readme more human-sensible at one line

### PR DESCRIPTION
**Pull request by aaarkid**
Line 141: Se/dese/rializing -> De/serializing

### CODE REVIEW

**Comment by aaarkid** [comment]
Just came upon this project by random and found this one line weird. Crate seems very cool!

File: README.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ features = ["event-stream"]
 | Feature        | Description                                  |
 |:---------------|:---------------------------------------------|
 | `event-stream` | `futures::Stream` producing `Result<Event>`. |
-| `serde`        | Se/dese/rializing of events.                 |
+| `serde`        | De/serializing of events.                    |
```
Comment (CenTdemeern1) [inline README.md:141]:
```suggestion
| `serde`        | Serializing/deserializing of events.         |
```

**Review by CenTdemeern1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CenTdemeern1) [reply README.md:141]: Another option:
```suggestion
| `serde`        | (De)serializing of events.                   |
```

**Comment by CenTdemeern1** [comment]
@aaarkid Could you please make the requested changes? The PR is getting stale.

**Comment by aaarkid** [comment]
Sorry, I wasn't aware my action was required. Committed the suggestion.

**Review by CenTdemeern1 — APPROVED** [review]
Thanks!

**Comment by CenTdemeern1** [comment]
@TimonPost ?

**Review by TimonPost — APPROVED** [review]
Cool!

**Pull request merged**